Pages/Index.cshtml.cs

```diff
@@ -27,15 +29,16 @@ public void OnGet()
 
             using (var webClient = new WebClient())
             {
-                // string neighborhood = GetData("https://cagisonline.hamilton-co.org/arcgis/rest/services/CINC_PLANNING/Munitwps_Neighborhoods/MapServer/1/query?where=1%3D1&outFields=OBJECTID,CAGIS.Cinc_Community_Council_Bnd.AREA,PERIMETER,NEIGH_BND_,NEIGH,LINE,ACRES,NEIGH_BOUN,REVISED_DA&returnGeometry=false&outSR=4326&f=json");
+                // string neighborhood = GetData("");
```

Comment:
Delete this entirely.  It's a no-no to commit commented code to VCS.  You can always go back in history to see what used to be there.